client/rpc_client.go

```diff
@@ -187,13 +187,14 @@ func (c *RPCClient) Close() error {
 
 // ForceLeave is used to ask the agent to issue a leave command for
 // a given node
-func (c *RPCClient) ForceLeave(node string) error {
+func (c *RPCClient) ForceLeave(node string, prune bool) error {
```

Comment:
Do we care about breaking callers of this function? If it's just consul that we expect to be using it then it's fine, otherwise you may need to introduce a new function with 2 args and delegate the old function to it.

Reply:
@s-christoff Change this to take a variadic argument of type `Option` instead. That will keep the change in signature from breaking downstream users.